### PR TITLE
JavaScript: Tabs

### PR DESCRIPTION
**Pull request by clrux**
This piece of work builds functionality for tabbed interfaces in the pattern library site. It includes JS and CSS additions as well as a few template updates.

@talbs, mind giving this one a review? Also feel free to customize the tab styles or ping me with suggestions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,52 +24,56 @@ <h1 class="pl-heading pl-element-title">{{ post.title }}</h1>
                 </ul>
```
Comment (talbs) [inline index.html]: Reminder: whatever layout changes are done here should also be done to the permalink ``layouts/element.html`` template as well.

Comment (talbs) [inline index.html]: I'm making a quick branch off of this and will sync these two views up.

**Comment by talbs** [comment]
:+1: - yay for UI simplification

**Comment by clrux** [comment]
@talbs Rebased with master, ready for merge.

**Pull request merged**